### DISCUSSION TITLE
Request does not execute when ky instance has prefixUrl

### DISCUSSION THREAD

**Discussion by jeepers3327** [sindresorhus/ky#346] [Q&A]
I created a single instance of `ky` using `ky.create`
```
const client = ky.create({
  credentials: 'include',
  prefixUrl: process.env.NEXT_PUBLIC_API_BASE_URL,
  headers: {
    'Content-Type': 'application/json',
  },
});
```
Then use that instance to perform authentication request to server
```
export const createUserSession = async (
  payload: LoginPayload,
): Promise<UserState> => {
  const data = await client
    .post('/session', {
      json: payload,
    })
    .json<UserState>();

  return data;
```

when the function is called, it seems that the AJAX request was not executed at all (AJAX request was not present in browser dev tools network tab), as I monitor the server and it endpoint was not called.

If I remove the prefixUrl from the instance, the request executed.
```
const client = ky.create({
  credentials: 'include',
  headers: {
    'Content-Type': 'application/json',
  },
});
```

Is there a configuration I needed to do inorder for the request to be executed when there's a prefixUrl set?

**Comment by sholladay** [comment]
My guess is that `process.env.NEXT_PUBLIC_API_BASE_URL` is being executed in the browser and that it's throwing an error because `process` is not available.

**Reply by jeepers3327** [reply]
Thank you for the response, according to NextJS docs, if I prepend my env variable with NEXT_PUBLIC, it will be exposed to the browser

**Reply by sholladay** [reply]
Is there a stack trace or error message anywhere?

**Reply by jeepers3327** [reply]
As far as I can remember, there are no errors printed in the terminal or browser logs. Will try to put up a reproducible repo later if it still persist on newly created project.

**Reply by sholladay** [reply]
Thanks, I'll take a look at your repo when it's ready.

**Reply by sholladay** [reply]
Hi, @jeepers3327. Just doing some housekeeping on the discussions here. Were you able to make a reproducible example for me to look at?

**Reply by jeepers3327** [reply]
Hi unfortunately I wasn't able to make an example. I 've used axios as a solution for it. I'm not sure if I can give you more information regarding since I can barely recall the project that I work at that time.

My apologies.

**Reply by sholladay** [reply]
Okay, no worries. We've done multiple releases since your question was posted, so it's possible the issue is fixed. I hope you have better luck in the future if you try Ky again.

**Reply by sholladay** [reply]
FYI, I think the problem here was that `/session` should be `session` when using `prefixUrl`.
